core/spec.md

```diff
@@ -2317,14 +2359,20 @@ and the following Resource specific attributes:
 and, finally, the following Version specific attributes:
 
 **`RESOURCEurl`**
-- Type: URL
+- Type: URI
 - Description: if the content of the Resource are stored outside of the
   current Registry then this attribute MUST contain a URL to the
-  location where it can be retrieved
+  location where it can be retrieved. If the value of this attribute
+  is a well-known identifier that is readily understood by all registry
+  users and resolves to a common representation of the Resource, or
+  an item in some private store/cache, rather than a networked document
+  location, then it is RECOMMENDED for the value to be a uniform resource
+  name (URN).
 - Constraints:
-  - REQUIRED if the Resource contents are stored outside of the current
+  - REQUIRED if the Resource contents are not stored inside of the current
     Registry
-  - the referenced URI MUST support an HTTP(s) `GET` to retrieve the contents
+  - if the contents are stored in a network accessible endpoint then the
+    referenced URL MUST support an HTTP(s) `GET` to retrieve the contents
```

Comment:
I'm confused as to the change from URI to URL here, given that the type of changing from URL to URI (and the name still uses URL...)

Reply:
Might need to tweak the wording, but it's defined as a URI to allow for just about anything - including something that's not network accessible... like a URN or some well known identifier. However, if it is network accessible then it needs to be a URL.  Does that make any sense?

Reply:
Let's discuss the URL vs URI in the name during the call... that's a good one :-)

Reply:
Do you think I need to change the text to make it clearer?

Reply:
Probably not. I wouldn't have spotted it at all if it weren't for the fact that it changed in one direction when the type changed in the opposite direction in the same PR.